src/qiskit_iqm/fake_backends/quantum_architectures/adonis.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2022 Qiskit on IQM developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Adonis is IQM's 5-qubit architecture with star topology: the central qubit is connected to the other four
+"""
+
+from .quantum_architecture import IQMQuantumArchitecture
+
+
+class Adonis(IQMQuantumArchitecture):
+    """IQM's five-qubit transmon device.
+
+    The qubits are connected thus::
+            QB1
```

Comment:
The syntax is broken here, Sphinx raises an error.
```suggestion

            QB1
```